src/backend/gporca/libgpopt/include/gpopt/mdcache/CMDCache.h

```diff
@@ -44,6 +44,13 @@ class CMDCache
 	// the maximum size of the cache
 	static ULLONG m_ullCacheQuota;
 
+	// if we have cached a relation without an index, because that index cannot
+	// be used in the current transaction (for more info see src/backend/access/heap/README.HOT),
+	// we save the transaction's xmin. If later TransactionXmin changes from
+	// the saved value, the cache will be reset and the relation will be
+	// reloaded with the index usage.
+	static uint32_t m_transientXmin;
```

Comment:
Why uint32_t instead TransactionId?

Reply:
it looks like we can't include anything from gpdb in CMDCache since the build takes place separately in unit tests.